docs/reference/job_definition.md

```diff
@@ -36,6 +36,9 @@ definition.
   values used by ClusterFuzz. The same type of variable is available for the
   other sanitizers, i.e. **ADDITIONAL_MSAN_OPTIONS** or
   **ADDITIONAL_UBSAN_OPTIONS** (see an example below on this page).
+* **USES_MTE**: indicates whether a device is fuzzing with the Memory Tagging
```

Comment:
@oliverchang friendly ping.

FWIW we are not trying to use the the job name to extract any info. We simply want to capture that the env variable `USES_MTE` is set to `True` to indicate to ClusterFuzz that we should proceed to build the fuzzing command accordingly.

We use this style for a slew of other env variables in prod, such as `RUN_TIMEOUT` and `OS_OVERRIDE`.

Reply:
Sorry for the slow reply! I think it may be better to be more consistent here, and support this like so: https://github.com/google/clusterfuzz/blob/22d30c3eaccce13631a1a38b5fed1b4e394cad0e/src/clusterfuzz/_internal/system/environment.py#L506 

as @jonathanmetzman suggests. 

The concept of "sanitizers" is separate from most other env vars, and adding more env vars is also no ideal.

Reply:
Done!

Instead of pulling from the job name, I refactored to use `android.settings.is_mte_build()`, which checks the connected device for MTE.